src/App.tsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import './App.scss';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [focusedQuery, setFocusedQuery] = useState(false);
```

Comment:
Rename all boolean variables using this [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isFocusedQuery, setIsFocusedQuery] = useState(false);
```